features/dd-sdk-android-logs/transitiveDependencies

```diff
@@ -2,7 +2,7 @@ Dependencies List
 
 androidx.annotation:annotation:1.3.0                            :   30 Kb
 com.google.code.gson:gson:2.10.1                                :  276 Kb
-org.jetbrains.kotlin:kotlin-stdlib-common:1.8.22                :  216 Kb
+org.jetbrains.kotlin:kotlin-stdlib-common:1.9.10                :  219 Kb
```

Comment:
you are updating to `1.9` here but in the prev PR you mentioned that this is not yet recommended. Are we safe here ?